.github/workflows/ci.yml

```diff
@@ -35,17 +37,25 @@ jobs:
         with:
           version: "${{ matrix.NXF_VER }}"
 
+      - name: Setup apptainer
+        uses: eWaterCycle/setup-apptainer@main
+
+      - name: Set up Singularity
+        run: |
+          mkdir -p $NXF_SINGULARITY_CACHEDIR
+          mkdir -p $NXF_SINGULARITY_LIBRARYDIR
+
       - name: Download test data
         # Download A fungal test data set that is full enough to show some real output.
         run: |
           curl https://tolit.cog.sanger.ac.uk/test-data/resources/treeval/TreeValTinyData.tar.gz | tar xzf -
 
-      - name: Run MAPS_ONLY pipeline with test data
+      - name: Docker - Run ALL_FILES pipeline with test data
         # Remember that you can parallelise this by using strategy.matrix
         run: |
-          nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results -entry MAPS_ONLY
+          nextflow run ${GITHUB_WORKSPACE} -profile test_full,docker --outdir ./Dock-res
```

Comment:
Why `test_full` ? It's `test` we're supposed to run on CI, so that it runs in minutes

Reply:
They are both the same at the minute, the only difference is that test has local paths and test_full has github paths. I can rename to make that more clear.

Reply:
I've changed this now. This uses the same dataset as TreeVal due to the similarity in processes, this also means we have the same limitation in shrinking the data. Any smaller and we wont be able to tell whether it is working.